source/reactions.yaml

```diff
@@ -0,0 +1,111 @@
+  '/teams/{team_id}/channels/{channel_id}/posts/{post_id}/save_reaction':

```

Comment:
If reactions are a new object/resource we should treat them like the other resources so maybe make the route this:
`/teams/{team_id}/channels/{channel_id}/posts/{post_id}/reactions/create`

With similar changes to the other ones